README.md

```diff
@@ -37,8 +37,11 @@ Every time an stratum job notification is received miner update its current work
 - 3D BOX [here](3d_files/)
 
 #### Current Supported Boards
-- TTGO T-Display S3 ([Aliexpress link](https://s.click.aliexpress.com/e/_Ddy7739))
-- ESP32-WROOM-32, ESP32-Devkit1.. ([Aliexpress link](https://s.click.aliexpress.com/e/_DCzlUiX))
+- TTGO T-Display S3 ([Aliexpress link*](https://s.click.aliexpress.com/e/_Ddy7739))
+- ESP32-WROOM-32, ESP32-Devkit1.. ([Aliexpress link*](https://s.click.aliexpress.com/e/_DCzlUiX))
+- ESP32-cam [Board Info](https://lastminuteengineers.com/getting-started-with-esp32-cam/)
+
+* affiliate link
```

Comment:
I agree on adding this, and board info for each one

Reply:
You could add to the donation section of the readme, something like ...or order your board by using one of the affiliate links above and even provide links to other marketplaces too. My goal was to link to a detailed description of the board, as it is not esp32-S3 based.